keyboards/gray_studio/space65r3/space65r3.c

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```